resources/js/components/forms/gallery-dialogs/DeleteDialog.vue

```diff
@@ -4,7 +4,9 @@
 			<div>
 				<p class="p-9 text-center text-muted-color max-w-xl text-wrap">
 					{{ confirmation }}<br />
-					<span class="text-warning-700"><i class="pi pi-exclamation-triangle mr-2" />{{ $t("dialogs.delete_album.warning") }}</span>
+					<span class="text-warning-700"
+						><i class="pi pi-exclamation-triangle ltr:mr-2 rtl:ml-2" />{{ $t("dialogs.delete_album.warning") }}</span
+					>
```

Comment:
Odd place to split lines?

Reply:
Fixed [bf6ffa1](https://github.com/LycheeOrg/Lychee/pull/3419/commits/bf6ffa1d6a3901fde86e6b58c01b5cc3cef62632)